test/Microsoft.SqlTools.ServiceLayer.UnitTests/QueryExecution/Common.cs

```diff
@@ -55,18 +68,53 @@ public static class Common
         public static readonly SelectionData SubsectionDocument = new SelectionData(0, 0, 2, 2);
 
         #endregion
+        public static TestResultSet ZeroRowTestResultSet => new TestResultSet(StandardColumns, ZeroRows);
+
+        public static TestResultSet[] ZeroRowTestDataSet => new[] { ZeroRowTestResultSet };
 
         public static TestResultSet StandardTestResultSet => new TestResultSet(StandardColumns, StandardRows);
 
-        private static readonly Lazy<TestResultSet> TestResultSet = new Lazy<TestResultSet>(new TestResultSet(StandardColumns, MillionRows));
+        public static TestResultSet[] StandardTestDataSet => new[] { StandardTestResultSet };
+
+
+        public static TestResultSet HundredRowTestResultSet => new TestResultSet(StandardColumns, HundredRows);
+
+        public static TestResultSet[] HundredRowTestDataSet => new[] { HundredRowTestResultSet };
+
+
+        public static TestResultSet ThousandRowTestResultSet => new TestResultSet(StandardColumns, ThousandRows);
+
+        public static TestResultSet[] ThousandRowTestDataSet => new[] { ThousandRowTestResultSet };
 
-        public static TestResultSet MillionRowTestResultSet => TestResultSet.Value;
+        private static readonly Lazy<TestResultSet> HundredKRowTestResultSetLazy = new Lazy<TestResultSet>(new TestResultSet(StandardColumns, HundredKRows));
 
-        private static readonly Lazy<TestResultSet[]> MillionRowResultSet = new Lazy<TestResultSet[]>(new[]{MillionRowTestResultSet});
+        public static TestResultSet HundredKRowTestResultSet => HundredKRowTestResultSetLazy.Value;
 
-        public static TestResultSet[] StandardTestDataSet => new [] {StandardTestResultSet};
+        private static readonly Lazy<TestResultSet[]> HundredKRowTestDataSetLazy = new Lazy<TestResultSet[]>(new[] { HundredKRowTestResultSet });
 
-        public static TestResultSet[] MillionRowTestDataSet => MillionRowResultSet.Value;
+        public static TestResultSet[] HundredKRowTestDataSet => HundredKRowTestDataSetLazy.Value;
+
+
+        private static readonly Lazy<TestResultSet> MillionRowTestResultSetLazy = new Lazy<TestResultSet>(new TestResultSet(StandardColumns, MillionRows));
+
+        public static TestResultSet MillionRowTestResultSet => MillionRowTestResultSetLazy.Value;
+
+        private static readonly Lazy<TestResultSet[]> MillionRowTestDataSetLazy = new Lazy<TestResultSet[]>(new[]{ MillionRowTestResultSet });
+
+        public static TestResultSet[] MillionRowTestDataSet => MillionRowTestDataSetLazy.Value;
+
+        public static IEnumerable<TestResultSet> TestResultSetsEnumeration
+        {
+            get
+            {
+                yield return StandardTestResultSet;
+                yield return  HundredRowTestResultSet;
+                yield return  ThousandRowTestResultSet;
+                yield return  ZeroRowTestResultSet;
+                yield return  HundredKRowTestResultSet;
+                yield return  MillionRowTestResultSet;
```

Comment:
minor: most of these have 2 spaces after `return`

Reply:
Fixed.

Reply:
fixed.